phpunit/block-supports/duotone-test.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+/**
+ * Test the block duotone support.
+ *
+ * @package Gutenberg
+ */
+
+class WP_Block_Supports_Duotone_Test extends WP_UnitTestCase {
+	public function tear_down() {
+		wp_clean_themes_cache();
+		unset( $GLOBALS['wp_themes'] );
+		WP_Style_Engine_CSS_Rules_Store_Gutenberg::remove_all_stores();
+		parent::tear_down();
+	}
```

Comment:
I'm working on getting tests running locally to see if any of this is necessary and, if so, why it is necessary.

Reply:
They didn't seem necessary.